templates/Datashades-OpsWorks-CKAN-Instances.cfn.yml.j2

```diff
@@ -198,8 +198,14 @@ Resources:
   OpswxEC2Solr1AmazonLinux2:
     Type: AWS::OpsWorks::Instance
     Properties:
-      Hostname: !Sub "${ApplicationId}-solr1{% if item.tags["PowerManaged"] == "Yes" %}-PM{% endif %}-al2"
+      Hostname: !Sub "${ApplicationId}-solr1{% if item.tags["PowerManaged"] == "Yes" %}-PM{% endif %}-al2b"
       RootDeviceType: ebs
+      BlockDeviceMappings:
+        - DeviceName: "ROOT_DEVICE"
+          Ebs:
+            DeleteOnTermination: true
+            VolumeSize: 100
```

Comment:
This should be a parameter, perhaps with a default value.

Reply:
any time this changes it will need a replacement of the entire instance.